sideboard.lua

```diff
@@ -32,7 +32,13 @@ function Sideboard:draw()
    love.graphics.setFont(self.font)
    love.graphics.setColor(0, 0, 0)
    love.graphics.print("Score " .. self.score, self.x, self.y)
-   love.graphics.print("Combo " .. self.combo .. "x", self.x, self.y + self.fontHeight)
+   if self.combo >= 10 then
+       love.graphics.setColor(1, 0.5, 0)
+       love.graphics.print("Combo " .. self.combo .. "x", self.x, self.y + self.fontHeight)
+       love.graphics.setColor(0, 0, 0)
```

Comment:
Du burde ikke sette fargen etter. Bedre at alle som er avhengig av en spesifikk farge setter det selv, for å unngå overraskende fargebytter.